src/index.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
+import './App.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
```

Comment:
- [ ] it would be better to remove all the boilerplate files and only keep the used ones in order to complete the requirements